group_project/assignment_1.py

```diff
@@ -1,6 +1,7 @@
 def first_half(a_string):
-    pass
-
+    halflength = int(round(len(a_string)/2.0))
```

Comment:
This is a really clever solution Michael!

Reply:
By creating the variable and using the round function you compile case for odd and even in one line. so resulting answer two lines. Great job!